src/reynir_correct/pattern.py

```diff
@@ -399,7 +633,9 @@ def wrong_preposition_að_leiðandi(self, match: SimpleTree) -> None:
     def wrong_preposition_að_mörkum(self, match: SimpleTree) -> None:
         """ Handle a match of a suspect preposition pattern """
         # Find the offending prepositional phrase
-        pp = match.first_match("PP > { 'að' ( 'mark'|'mörk' ) }", self.ctx_að)
+        pp = match.first_match("PP > { 'að' 'mark' }")
```

Comment:
This might actually work better with `pp = match.first_match('PP > { "að" "mörkum" }')`, i.e. using double quotes for a literal match, instead of single quotes for a lemma match.